zuul.d/jobs.yaml

```diff
@@ -1122,6 +1122,19 @@
     nodeset: iosxr-6.1.3-python27
     vars:
       ansible_test_python: 2.7
+      ansible_test_split_in: 2
+
+- job:
+    name: ansible-test-network-integration-iosxr-network_cli-python27_1_of_4
```

Comment:
Lets rename to ansible-test-network-integration-iosxr-network_cli-python27-scenario001